pkg/container-hook/tracer.go

```diff
@@ -47,7 +47,6 @@ import (
 
```

Comment:
I don't think we should this. Instead, we should deprecate and delete runc-fanotify in favour of container-hook https://github.com/inspektor-gadget/inspektor-gadget/issues/1831.